### ISSUE TITLE
UI enhancements

### ISSUE THREAD

**Issue by TeaSitta** [TeaSitta/AI-Horde-Worker#3]
-Add currently set context window to UI (near current threads, make sure it refreshes w/config refresh like threads does)

-Add the horde queue for the hosted model (under 'Entire Horde' section)

-Runtime argument or config to set the maximum number of GPUs displayed

**Comment by TeaSitta** [comment]
Added context and model stats from the horde, waiting on Horde API PR https://github.com/Haidra-Org/AI-Horde/pull/391 before commiting to main

**Comment by TeaSitta** [comment]
Moving GPU display option to the backburner on another issue so I can close this.
